### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by M3t0r**
Looks like this:
Screenshot 2020-08-20 at 20 12 42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ Thanks for considering to contribute to this project. Here's how to get started:
 
     ```shell
     cd website/
-    gatsby develop
```
Comment (mre) [inline README.md:35]: No ❤️ for Gatsby anymore?

**Review by mre — APPROVED** [review]

**Comment by mre** [comment]
Your changes look good. No clue why CI is failing. 🤔

File: README.md (hunk 1 above)
Comment (M3t0r) [reply README.md:35]: I never had any love for Gatsby. But also, `yarn develop` calls the Gatsby binary from `node_modules/`.

**Comment by M3t0r** [comment]
> Your changes look good. No clue why CI is failing. 🤔

The secrets for the search service are missing. I also can't build it locally. Only the `develop` function works.

File: README.md (hunk 1 above)
Comment (mre) [reply README.md:35]: Ah makes sense.

**Comment by mre** [comment]
> The secrets for the search service are missing. 

I'm confused. The secrets should be there.
https://github.com/analysis-tools-dev/website/blob/ca027868dbccc64249b6d334312d8760a04ead4d/.github/workflows/ci.yaml#L40-L48

and
image

Also we haven't touched them and the last PR went through (https://github.com/analysis-tools-dev/website/pull/9).
The only thing that changed lately was the migration from the free plan to the open source plan, but that shouldn't have invalidated the tokens.

**Comment by mre** [comment]
Local build works fine. Now I'm even more confused.
Will recreate the secrets.

**Comment by M3t0r** [comment]
This is what I get locally:
```
simon:analysis-tools-website/$ git rev-parse HEAD
63a0dbcb38144c9f4a7de1ee1f28b70d168cb5ff
simon:analysis-tools-website/$ yarn build
yarn run v1.22.4
$ gatsby build
success open and validate gatsby-configs - 0.187s
success load plugins - 5.335s
Loaded gatsby-starter-plugin
success onPreInit - 0.196s
success delete html and css files from previous builds - 0.141s
success initialize cache - 0.007s
success copy gatsby files - 3.043s
Fetching voting data from the GCP endpoint
success onPreBootstrap - 1.191s
success createSchemaCustomization - 0.007s
success Checking for changed pages - 0.007s
success source and transform nodes - 40.626s
success building schema - 0.505s
success createPages - 0.680s
success Checking for changed pages - 0.001s
success createPagesStatefully - 0.109s
success update schema - 0.047s
success onPreExtractQueries - 0.002s
success extract queries from components - 2.939s
success write out redirect data - 0.004s
success onPostBootstrap - 0.001s
info bootstrap finished - 104.233s
success run page queries - 6.076s - 565/565 92.99/s
success write out requires - 0.014s
success Building production JavaScript and CSS bundles - 16.950s
success Rewriting compilation hashes - 0.031s
success Building static HTML for pages - 5.031s - 568/568 112.90/s

 ERROR #11321  PLUGIN

"gatsby-plugin-algolia" threw an error while running the onPostBuild lifecycle:

Please provide an application ID. Usage: algoliasearch(applicationID, apiKey, opts)

  49 |   activity.start();
  50 |
> 51 |   const client = algoliasearch(appId, apiKey);
     |                  ^
  52 |
  53 |   setStatus(activity, `${queries.length} queries to index`);
  54 |

File: node_modules/gatsby-plugin-algolia/gatsby-node.js:51:18



  AlgoliaSearchError: Please provide an application ID. Usage: algoliasearch(app  licationID, apiKey, opts)

  - AlgoliaSearchCore.js:50 AlgoliaSearchNodeJS.AlgoliaSearchCore
    [analysis-tools-website]/[gatsby-plugin-algolia]/[algoliasearch]/src/Algolia    SearchCore.js:50:11

  - AlgoliaSearch.js:11 AlgoliaSearchNodeJS.AlgoliaSearch
    [analysis-tools-website]/[gatsby-plugin-algolia]/[algoliasearch]/src/Algolia    Search.js:11:21

  - AlgoliaSearchServer.js:17 AlgoliaSearchNodeJS.AlgoliaSearchServer
    [analysis-tools-website]/[gatsby-plugin-algolia]/[algoliasearch]/src/server/    builds/AlgoliaSearchServer.js:17:17

  - node.js:83 new AlgoliaSearchNodeJS
    [analysis-tools-website]/[gatsby-plugin-algolia]/[algoliasearch]/src/server/    builds/node.js:83:23

  - node.js:68 algoliasearch
    [analysis-tools-website]/[gatsby-plugin-algolia]/[algoliasearch]/src/server/    builds/node.js:68:10

  - gatsby-node.js:51 Object.exports.onPostBuild
    [analysis-tools-website]/[gatsby-plugin-algolia]/gatsby-node.js:51:18

  - api-runner-node.js:330 runAPI
    [analysis-tools-website]/[gatsby]/src/utils/api-runner-node.js:330:22

  - api-runner-node.js:440 Promise.catch.decorateEvent.pluginName
    [analysis-tools-website]/[gatsby]/src/utils/api-runner-node.js:440:17

  - debuggability.js:384 Promise._execute
    [analysis-tools-website]/[bluebird]/js/release/debuggability.js:384:9

  - promise.js:518 Promise._resolveFromExecutor
    [analysis-tools-website]/[bluebird]/js/release/promise.js:518:18

  - promise.js:103 new Promise
    [analysis-tools-website]/[bluebird]/js/release/promise.js:103:10

  - api-runner-node.js:439
    [analysis-tools-website]/[gatsby]/src/utils/api-runner-node.js:439:14

  - util.js:16 tryCatcher
    [analysis-tools-website]/[bluebird]/js/release/util.js:16:23

  - reduce.js:166 Object.gotValue
    [analysis-tools-website]/[bluebird]/js/release/reduce.js:166:18

  - reduce.js:155 Object.gotAccum
    [analysis-tools-website]/[bluebird]/js/release/reduce.js:155:25

  - util.js:16 Object.tryCatcher
    [analysis-tools-website]/[bluebird]/js/release/util.js:16:23


not finished onPostBuild - 0.308s
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by mre** [comment]
Odd. Triggered a build on master today and that seemed to work. Let's see what happens  if we merge this.
Thanks for the improvements btw.

**Pull request merged**